### PR TITLE
Fixing KEGG colored pathway

### PR DESCRIPTION
**Pull request by cokelaer**

### CODE REVIEW

File: sequana/enrichment/kegg.py [hunk 1]
```diff
@@ -110,6 +113,19 @@ class KEGGPathwayEnrichment:
         ke.scatterplot('up')
         savefig("B4052_T1vsT0_KE_scatterplot_up.png")
 
+
+    Pathways are loaded from KEGG which may take some time. For development or production, you may want
+    to save the KEGG pathways is a given place. Note however, that the pathways that are enriched will
```
Comment (rania-o) [inline sequana/enrichment/kegg.py:118]: in* a given place

File: sequana/enrichment/kegg.py [hunk 2]
```diff
@@ -110,6 +113,19 @@ class KEGGPathwayEnrichment:
...
+    still be downloaded for the final annotation and image creation
+
+    To save the pathways locally to load them later do as follows (here for human)::
```
Comment (rania-o) [inline sequana/enrichment/kegg.py:121]: and* load..

**Review by rania-o — APPROVED** [review]

File: sequana/modules_report/kegg_enrichment.py [hunk 3]
```diff
@@ -116,10 +129,12 @@ def summary(self):
 <p> In the following plots you can find the first KEGG Pathways that are enriched, keeping a 
 maximum of {self.nmax} pathways. </p>
 
-<p>The KEGG name used is {self.organism}.<br>
-
+<p>The KEGG name used is <b>{self.organism}</b>.
 
+<p>This table gives you (column N) the number of genes used when performing the enrichment on up-regulated genes (up), down-regulated genes, and when we take all up and down regulated genes (all). We also provide the total number of genes that were analysed in the differential analysis (all genes). This number may be lower that the number of genes found in the annotation if a filter was applied. Finally, using those genes, the provide the percentage of those genes that were found in the KEGG database. the Enrichment analysis relies on a background. This background was defined as {self.ke.background} </p>
```
Comment (khourhin) [inline sequana/modules_report/kegg_enrichment.py:134]: Finally, using those genes, "the" > "we"

File: sequana/scripts/main/enrichment_kegg.py [hunk 4]
```diff
@@ -87,7 +95,8 @@
     "--kegg-background",
     type=click.INT,
     default=None,
-    help="""a background for kegg enrichment. If None, set to number of genes found in KEGG""",
+    help="""a background for kegg enrichment. If None, set to the number of genes 
+used in the differential analysis (input file rnadfiff.csv).""",
```
Comment (khourhin) [inline sequana/scripts/main/enrichment_kegg.py:99]: rnadfiff > rnadiff

**Review by khourhin — APPROVED** [review]
BIg PR ;)

File: sequana/enrichment/kegg.py (hunk 1 above)
Comment (cokelaer) [reply sequana/enrichment/kegg.py:118]: fixed in next PR

**Pull request merged**